### DISCUSSION TITLE
Документация по организации разработки.

### DISCUSSION THREAD

**Discussion by LostAlly** [totumonline/totum-ru-issues-and-discussions#405] [Как это сделать?] [answered]
В документации есть раздел в котором на сколько я понял есть раздел как организовать dev/test и prod сервер, как объединять разработку с работающей системой но страница пуста, есть ли какие то сроки заполнения данной страницы, мне кажется она очень важна для начинающих, кто пытается одновременно изучать totum и и реализовывать свою задумку.
Или может можно описать это парой предложений в какую сторону копать дальше будем разбираться.

**Comment by erpic-ru** [comment, marked as answer]
https://ru.docs.totum.online/export#test-prod

**Reply by LostAlly** [reply]
Спасибо большое, я прочитал документацию хоть и по диагонали, помню,что такое раздел есть, но засомневался, найдя страницу https://ru.docs.totum.online/linking-the-test-environment-to-the-prod-base )) 
Прошу разработчиков установить в этом разделе ссылку на указанный в ответе ) хотя бы временно даже если планируется написание более подробной инструкции.
Спасибо.
